native~/Runtime/src/CesiumMetadataValueImpl.cpp

```diff
@@ -55,21 +56,22 @@ namespace CesiumForUnityNative {
 
 namespace {
 
-CesiumMetadataValueImpl::ValueType getNativeBooleanValue(
-    const DotNet::CesiumForUnity::CesiumMetadataValue& value) {
+template <typename F>
+auto getNativeBooleanValue(
+    const DotNet::CesiumForUnity::CesiumMetadataValue& value,
+    F&& callback) {
   std::optional<bool> maybeBoolean =
       CesiumForUnity::CesiumMetadataValue::GetObjectAsBoolean(
           value.objectValue());
   if (maybeBoolean) {
-    return *maybeBoolean;
+    return callback(*maybeBoolean);
```

Comment:
The other `get` functions use `std::move()` but this one omits it, so I just wanted to check if that was intended. (Perhaps a boolean is trivial enough that it's not necessary?)

Reply:
Right, doesn't make any sense to move something that's known to be a bool.